conf/restperf/9.12.0/disk.yaml

```diff
@@ -45,7 +45,43 @@ plugins:
   - Max:
       - node<>node_disk_max
       - aggr<>aggr_disk_max ...
-  - Disk
+
+  - Disk:
+      objects:
+        - fans => fan:
+            - ^^id => fan_id
```

Comment:
probably better to align on `=>` like we do in the counters section

Reply:
Done